test/authNZ.mjs

```diff
@@ -28,9 +32,14 @@ describe.skip("authNZMiddleware", function () {
   };
 
   beforeEach(() => {
+    const __filename = fileURLToPath(import.meta.url);
+    const __dirname = path.dirname(__filename);
+    const homedir = path.join(__dirname, "../../test/test-homedir");
+
     container = setupContainer();
     container.register({
       accountClient: awilix.asFunction(mockAccountClient).scoped(),
+      homedir: awilix.asFunction(() => homedir).scoped(),
```

Comment:
if this is a good general-purpose default, you can move it up to `./src/config/setup-test-container.mjs`.